auth/src/main/java/com/firebase/ui/auth/ui/AcquireEmailHelper.java

```diff
@@ -80,30 +80,22 @@ private void startEmailHandler(String email, List<String> providers) {
             mActivityHelper.startActivityForResult(registerIntent, RC_REGISTER_ACCOUNT);
         } else {
             // account does exist
-            for (String provider : providers) {
```

Comment:
Here's the most important change in this PR: the for loop doesn't loop! Is my change appropriate or should we be doing something else here?